### DISCUSSION TITLE
Is it possible to instantiate a webview (html) component with iced?

### DISCUSSION THREAD

**Discussion by stephane-klein** [hecrj/iced#1740] [Q&A]
Hello,

If for example I want to develop a mail client with *iced*, is it possible to instantiate a "webview" component? For example to display the html content of a email?

If yes, do you have a small example of that allows to instantiate a webview component? For example, based on WebKit (or [Servo](https://github.com/servo/servo/) 😉) ?

Best regards,
Stéphane

**Comment by 13r0ck** [comment]
no
